crates/primitives/benches/bench.rs

```diff
@@ -43,7 +43,7 @@ fn bench_key_add_assign_u64(c: &mut Criterion) {
     c.bench_function("Key2::add_assign(u64)", |b| {
         b.iter(|| {
             let mut copy = black_box(key);
-            let _ = black_box(copy += 1u64);
+            let _ = black_box(|| copy += 1u64);
```

Comment:
What lint is this from?

Reply:
https://github.com/bheisler/criterion.rs/issues/507. I'm also not happy with this hack, since there is some non-intended overhead with this closure.